app/frontend/src/features/search/SearchResultsList.tsx

```diff
@@ -44,11 +49,17 @@ const useStyles = makeStyles((theme) => ({
     },
   },
   searchResult: {
-    [theme.breakpoints.down("sm")]: {
-      maxWidth: "80%",
-    },
-    [theme.breakpoints.up("md")]: {
-      margin: theme.spacing(2),
+    marginBottom: theme.spacing(3),
+    borderRadius: theme.shape.borderRadius * 2,
+    backgroundColor: theme.palette.primary.contrastText,
+    boxShadow: "0 0 4px rgba(0,0,0,0.25)",
```

Comment:
I think we should either use `elevation` on the card, or set this in the theme somewhere

Reply:
Box shadow in MUI works only accepting parameters. So we cannot just set that boxShadow in the theme file, as it gives an error. The alternative would be to store it as a CSS variable (we can place that in the index.css file).

Reply:
You can put what you like in the theme file (even just this whole string) if you extend the type definition (as is being done at the top of `theme.ts` for `theme.shape.borderRadius`)